lib/server.js

```diff
@@ -168,7 +169,11 @@ Server.prototype._start = function (config, launcher, preprocess, fileList,
 
   var afterPreprocess = function () {
     if (config.autoWatch) {
-      self._injector.invoke(watcher.watch)
+      if (config.watchMode == 'ibazel') {
```

Comment:
It may be write within single line of code: `self._injector.invoke(config.watchMode == 'ibazel' ? ibazel_watcher.watch : watcher.watch)`